sender-serice/cmd/sender-service/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	server "github.com/DarkJediDJ/support-chat/sender-serice/api"
+	"github.com/segmentio/kafka-go"
+)
+
+const addr = ":8080"
+
+func main() {
+	a := server.App{}
+
+	conn, err := kafka.DialLeader(context.Background(), "tcp", "localhost:9092", "user-messages", 0)
+	if err != nil {
+		if err != nil {
+			log.Printf("An Error Occured %v", err)
+		}
+	}
```

Comment:
Why you check `err` 2 times?
Also need to exit program in this way, not only log error message.